src/Arcus.Observability.Tests.Unit/Telemetry/ILoggerExtensionsTests.cs

```diff
@@ -85,22 +85,27 @@ public void LogSqlDependency_ValidArguments_Succeeds()
             string tableName = _bogusGenerator.Name.FullName();
             string operationName = _bogusGenerator.Name.FullName();
             bool isSuccessful = _bogusGenerator.Random.Bool();
-            DateTimeOffset startTime = _bogusGenerator.Date.PastOffset();
-            var duration = _bogusGenerator.Date.Timespan();
-
-            // Act
-            logger.LogSqlDependency(serverName, databaseName, tableName, operationName, isSuccessful, startTime, duration);
 
-            // Assert
-            var logMessage = logger.WrittenMessage;
-            Assert.StartsWith(MessagePrefixes.DependencyViaSql, logMessage);
-            Assert.Contains(serverName, logMessage);
-            Assert.Contains(databaseName, logMessage);
-            Assert.Contains(tableName, logMessage);
-            Assert.Contains(operationName, logMessage);
-            Assert.Contains(isSuccessful.ToString(), logMessage);
-            Assert.Contains(startTime.ToString(CultureInfo.InvariantCulture), logMessage);
-            Assert.Contains(duration.ToString(), logMessage);
+            using (var measurement = DependencyMeasurement.Start(operationName))
```

Comment:
We need both approaches to be tested actually

Reply:
Ok